app/backend/controllers/profile.js

```diff
@@ -213,3 +216,22 @@ module.exports.rejectRequest = async (request, response) => {
   })
 }
 
+/*
+  Get method for articles of user.
+*/
+module.exports.getArticles = async (request, response) => {
```

Comment:
do we call this method?

Reply:
At some point, if someone needs a specific article, he/she can use this method.